assets/admin/tour/components/sensei-tour-kit/index.js

```diff
@@ -24,9 +24,16 @@ import { WpcomTourKit } from '@automattic/tour-kit';
  * @param {string}     props.tourName         The unique name of the tour.
  * @param {string}     props.trackId          ID of tracking event (optional). Tracking will be enabled only when provided.
  * @param {TourStep[]} props.steps            An array of steps to include in the tour.
+ * @param {Function}   [props.beforeEach]     A function to run before each step.
```

Comment:
It adds a new feature to run something before each step. Similar to what we have in tests to reset or prepare something. In the future, if needed it could have some more events like this.

Reply:
Nice!